### ISSUE TITLE
🐛 BUG: Browser history navigation does not work if an alert is open

### ISSUE THREAD

**Issue by atom-sgt** [dailydotdev/daily#1405]
### What went wrong? 🤔

I opened an item in my feed and clicked on the account that posted it to navigate to the account page.  I hit Follow on the account.  This opened an alert at the top of the page telling me that I had followed the account.  I then hit the back button expecting to go have to the previous screen.  The url changed, but the page did not.

### Expected Behavior

I expect to return to the previous page in the history and for the page to accurately reflect the current url.

### Steps to Reproduce Issue

```bash
1. Go to home page (https://app.daily.dev/)
2. Navigate to the account page of any post source (e.g. https://app.daily.dev/sources/hn)
3. Follow or unfollow the account to cause an alert to show
4. While the alert is open, click the browser back button to navigate back one page
5. Observe the url change back to the previous page (https://app.daily.dev/)
6. Observe that you are still on the account page
```

### Environment

Win10, Firefox 129.0.2

### Browsers

Firefox

### OS

Windows

### Code of Conduct

- [X] I follow the conditions of this project.

**Comment by idoshamun** [comment]
Hi @atom-sgt,

Unfortunately, I wasn't able to replicate your issue.
Are you able to do so constantly?

**Comment by rebelchris** [comment]
@atom-sgt Wondering if you still experience this issue?

**Comment by atom-sgt** [comment]
Yes, problem still exists, and yes I am able to replicate it consistently with the provided steps.

**Comment by atom-sgt** [comment]
Testing more, navigation appears to proceed once the notification has timed out.  I will create a video demonstrating this.  

Navigation through the history stack is actually happening, and it is possible to move through multiple urls, but daily.dev itself will not update until the notification expires.

**Comment by atom-sgt** [comment]
Video demonstration of navigation bug.
https://youtu.be/ruo_1cWQkH8

**Comment by rebelchris** [comment]
@atom-sgt Wow thanks that video helps a lot!
We'll look into it.

**Comment by sshanzel** [comment]
I think this may have to do with the recent change we did where we introduced `useTransition` which can be found in `useTimedAnimation`.

**Comment by LongTangGithub** [comment]
I tried to replicate this issue on my end also, and it seems fine. Was wondering if this fixed yet ?
